### DISCUSSION TITLE
Is it possible that cite/link to some blocks in other notes preciesly?

### DISCUSSION THREAD

**Discussion by zhang-stephen** [TriliumNext/Trilium#3351] [Ideas] [answered]
For example, I want to create link to a block, or a paragraph, of other text note, instead of the other note itself. 

Another scenario is, I want to insert mermaid notes or canvas notes and make them show the graph, instead of the URL itself, It could be used to generate something and to share them to other people.

It looks like obsidian or LogSeq, and I think precise citation is more useful and powerful for personal knowledge storage.

So is there someway to? or is there a widget to do this?

**Comment by zadam** [comment, marked as answer]
Hi, see https://github.com/zadam/trilium/issues/1021

The other thing you mention with mermaid kind of sounds like https://github.com/zadam/trilium/wiki/Text-notes#include-note

**Reply by zhang-stephen** [reply]
Well, looking forwarding to the link to paragraph...

As for including note, the included one cannot be rendered in shared page, even the included one is shared?

**Reply by zadam** [reply]
No, that's not possible. Sharing supports only a limited subset of the "full" trilium.

**Reply by zhang-stephen** [reply]
Expecting a more powerful sharing feature in trilium...
